lib/ansible/playbook/helpers.py

```diff
@@ -277,28 +254,7 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
                         variable_manager=variable_manager,
                     )
 
-                    # FIXME: remove once 'include' is removed
-                    # pop tags out of the include args, if they were specified there, and assign
```

Comment:
This is an extension of https://github.com/ansible/ansible/pull/74780 so I didn't add an extra clog frag, as it was included in the referenced PR.